lib/dai-plugin-mcd/src/ManagedCdp.js

```diff
@@ -32,6 +30,78 @@ export default class ManagedCdp {
     return art.times(rate).shiftedBy(-27);
   }
 
+  async getCollateralizationRatio() {
+    const [collateral, debt, price] = await Promise.all([
+      this.getCollateralValue(),
+      this.getDebtValue(),
+      this._cdpType.getPrice()
+    ]);
+
+    return (
+      collateral
+        .times(price)
+        .div(debt)
+        .toNumber() * 100
+    ).toFixed(2);
+  }
+
+  async getLiquidationPrice() {
+    const [liquidationRatio, debt, collateral] = await Promise.all([
+      this._cdpType.getLiquidationRatio(),
+      this.getDebtValue(),
+      this.getCollateralValue()
+    ]);
+    return debt
+      .times(liquidationRatio)
+      .div(collateral)
+      .toNumber()
```

Comment:
so here as well we would drop `.toNumber().toFixed()`, and then the return value will be a USD/COL CurrencyRatio, because we're doing DAI * (USD/DAI) / COL.

Reply:
ok that makes sense